src/CSharp/CodeCracker/Performance/RemoveWhereWhenItIsPossibleCodeFixProvider.cs

```diff
@@ -41,7 +41,7 @@ public sealed override FixAllProvider GetFixAllProvider()
             var whereMemberAccess = whereInvoke.ChildNodes().OfType<MemberAccessExpressionSyntax>().FirstOrDefault();
             var nextMethodMemberAccess = nextMethodInvoke.ChildNodes().OfType<MemberAccessExpressionSyntax>().FirstOrDefault();
             var newNextMethodInvoke = SyntaxFactory.InvocationExpression(
-                SyntaxFactory.MemberAccessExpression(SyntaxKind.SimpleMemberAccessExpression, whereMemberAccess.Expression, nextMethodMemberAccess.Name),
+                SyntaxFactory.MemberAccessExpression( SyntaxKind.SimpleMemberAccessExpression, whereMemberAccess.Expression, nextMethodMemberAccess.Name),
```

Comment:
Sorry. I should have audited the checkin a bit closer. Didn't intend to update any .cs files in this update. I did notice some potential issues/gaps in some of the CS scenarios, but didn't make modifications to them yet.